config/gaya.json

```diff
@@ -0,0 +1,12 @@
+{
+    "hostname": "gaya",
+    "active":true,
+    "execution_policy": "serial",
+    "exclusive_access":false,
+    "valid_systems":["*"],
+    "valid_prog_environs":["*"],
+    "launch_options":["-bind-to core"],
+    "omp_num_threads":1,
+    "reframe_base_dir":"/data/home/cladellash/benchmarking/build/reframe",
+    "reports_base_dir":"/data/home/cladellash/benchmarking/reports/"
```

Comment:
We could impose the `output/`, `perflogs/` and `stage/` reframe directories, as well as the reports directory to be relative to the repository location. Or simply fix them and remove and don't leave the choice. 
However, I'm afraid that it could cause some permission related issues on some systems.  What do you think ?